workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,200 @@
+| __Aggregated By__    | Groups all values in selected timerange for given metric __and/or__ attribute |
+
+#### Aggregate By
+
+The Aggregate By field is a nested aggregation technique that allows you to aggregate your initial query, the inner layer of data. The below example is grouping the `container.memory.percent` metric by averaging the values of the `container.name` attributes and summing `container.memory.percent`:
+
+![](https://lh7-us.googleusercontent.com/ohE5zbuYTvxfE_A3tAdR-ccxs11a_Z8XBPEmcDOLn4-jEe7pFGIwtOQDPkm4OwTH6h1-hmEUGZpFltgzQ1i2CwOn8gugkjRmIC4GM_YcFqGmnrCh1PBtHlFteYKpwiDSXCz_EZMfckZp-wbqhV-pWx4)
+
+#### Filters
+
+The filter feature is used to remove unwanted metrics within your query. The list of available filters is pulled from each metric's attributes, and the values are based on the data pulled for that attribute in that timerange. The below example is filtering out any `host.id` containing `ip-172-31-13-218.us-east-2.compute.internal`:
+
+![](https://lh7-us.googleusercontent.com/_50CgdVxjgsqeqS6U3S8Rud9llopVXSi97d2WXCo4shRYJp79HjUs8gGTq_ZLihDDlP4wpi3IqCvaT6z4NpnhXAQWzhNWbe2uUS89Gw4Nd8ww_RX95Ce5p4LsR_SBO3-0LSp3sAht3heL01QDKgCAQQ)
+
+The following table displays all available filter functions:
+
+| __Filter__                    | __Description__                                                |
+| :-------------------          | :------------------------------------------------------------ |
+| `!=`                          | Not Equal to |
+| `=`                           | Equal to |
+| `IN`                          | Checks if value is `IN` collection of filter values |
+| `NOT IN`                      | Checks if value is __not__ `IN` collection of filter values |
+| `LIKE`                        | Checks if value __and__ similar values in collection of filter values |
+| `NOT LIKE`                    | Checks if value __and__ similar values __not__ in collection of filter values |
+| `ILIKE (case insensitive)`    | Same as `LIKE` but case insensitive|
+| `NOT ILIKE (case insensitive)`| Same as `NOT LIKE` but case insensitive |
+| `IS NULL`                     | Checks if value in collection of filter values `IS NULL` |
+| `IS NOT NULL`                 | Checks if value in collection of filter values `IS NOT NULL` |
+
+#### Alias
+
+The Alias feature allows you to overwrite the metric name displayed in your widget for increased readability:
+
+![](https://lh7-us.googleusercontent.com/8gAYYTNk5B9dH_6a6S6DLq-zItBxQU_pv0GBuM0PnOonqY9kbxMiESmKRNa_vG6R7vDrhl5ELX_nRGloIlsHm5VDBM1xRuUuO7Az8wZk3orJnG6yrvTGeA8Nul_XDGugYy4_NnEavJISDxFtMdcMVyw)
+
+
+### Advanced
+
+#### Time Aggregation
+
+The Advanced section allows you to specify the interval at which your data is ingested and displayed.
+
+<Note> The default Data Aggregation Interval is assumed to be every 15 seconds unless specified. </Note>
+
+#### Unit Format
+
+The following table displays all available Unit Formats:
```

Comment:
Need one more sentence on what unit format does, e.g. "The Unit Format dictates how the metric is displayed on the frontend. This will impact the values you see on the y-axis as well as in the hover over tooltip.

The following table displays all available Unit Formats:"